manifests/init.pp

```diff
@@ -124,6 +125,7 @@
   $num_procs                             = $facts['processors']['count'],
   $install_method                        = $vault::params::install_method,
   $config_dir                            = if $install_method == 'repo' and $manage_repo { '/etc/vault.d' } else { '/etc/vault' },
+  $manage_package                        = true,
```

Comment:
Can you please add a unit test that sets this to false?

Reply:
```suggestion
  Boolean $manage_package                = true,
```
I know some of the parameters don't have a type when they should but other do already have types.